### PR TITLE
LPS-121908 Migrate management toolbar v2 usages in sharing-web

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
The <clay:management-toolbar-v2 /> tag has been replaced with the
<clay:management-toolbar /> tag (which uses clay v3).

**Test plan:** 
1. Create a new user
2. Go to D&M and "Share" an entry with that user
3. Log in with the new user
4. From the user's menu (top right user's icon), select "Shared Content"

/cc @markocikos @julien @jonmak08

Notes: previously discussed [here](https://github.com/liferay-lima/liferay-portal/pull/1509)

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
Hey @ambrinchaudhary could you please set the people you want to review the pr as reviewers instead of assigning? (Assign is meant to mark who is working on it). Thx!!

**Comment by boton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp [hunk 1]
```diff
@@ -106,22 +121,8 @@ viewSharedAssetsDisplayContext.populateResults(sharingEntriesSearchContainer);
...
-		).build()
-	%>'
-	module="SharedAssets.es"
-/>
\ No newline at end of file
+<aui:script>
+	Liferay.on('sharing:changed', () =>
+		Liferay.Portlet.refresh('#p_p_id<portlet:namespace />')
+	);
+</aui:script>
```
Comment (boton) [inline modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp:128]: Would it be better in a separate file?

Maybe with dispose method returned

```
return {
  dispose: () => Liferay.deatch(...
}
```

I read the previous discussion maybe in a new one or in the old SharedAssets.es.js, not in the SharedAssetsManagementToolbarPropsTransformer.js.

**Review by boton — COMMENTED** [review]
LGTM tested and working, I add one comment in inlined js but it is a minor issue.

@carloslancha feel free to forward it.

File: modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp (hunk 1 above)
Comment (carloslancha) [reply modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp:128]: Ideally we don't want to keep `js` code in `jsp`s, but **introducing** new js code in a `jsp` is definitely something we must not do.

File: modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/js/SharedAssetsManagementToolbarPropsTransformer.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	return {
+		...otherProps,
+		onFilterDropdownItemClick(event, {item}) {
+			const action = item.data?.action;
+
+			if (action === 'openAssetTypesSelector') {
+				openSelectionModal({
+					onSelect: (selectedItem) => {
+						if (selectedItem) {
+							let uri = viewAssetTypeURL;
```
Comment (carloslancha) [inline modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/js/SharedAssetsManagementToolbarPropsTransformer.js:31]: Can't we just do this?

```
const uri = addParams(
	`${portletNamespace}className=${selectedItem.value}`,
	viewAssetTypeURL
);
```

**Review by carloslancha — CHANGES_REQUESTED** [review]
Hey, I left a couple of comments, thx!

File: modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp:128]: @carloslancha do you agree with @boton 's proposal? Any other suggestion @julien @markocicos?

File: modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/js/SharedAssetsManagementToolbarPropsTransformer.js (hunk 2 above)
Comment (ambrinchaudhary) [reply modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/js/SharedAssetsManagementToolbarPropsTransformer.js:31]: You are right, I've just copied it but I'll change that

File: modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp (hunk 1 above)
Comment (carloslancha) [reply modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp:128]: I'd create a `SharedAssets` component to put it there, yup

Comment (ambrinchaudhary) [reply modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp]: done in https://github.com/liferay-lima/liferay-portal/pull/1548/commits/89ccb0b567f0bf783f062f0580525810d5e0fbcb

Comment (boton) [reply modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp]: LGTM, Ready to ship.
What do you think @carloslancha?

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
ci:forward

**Pull request closed without merging**